src/schema/portal/jgi_metagenomics.yaml

```diff
@@ -127,11 +127,12 @@ slots:
     recommended: true
   dna_cont_well:
     name: dna_cont_well
-    title: DNA well number
+    title: DNA plate position
     comments:
-    - Required when 'plate' is selected for container type. Corner wells must be blank/empty (A1, A12, H1, H12).
-      For partial plates, fill by columns. Leave blank if the sample will be shipped
-      in a tube.
+    - Required when 'plate' is selected for container type.
+    - Leave blank if the sample will be shipped in a tube.
```

Comment:
Will the sheet validate if this is left blank? Previously, we needed to put NA in this column because it's required. SO, tubes, put NA .. Plates, put the well number.

Reply:
Mark: we can use linkML rules. This adjusts slot validation based on the input of another slot. Is this possible with DH? Can DH implement this rule?